src/test/java/com/iexec/commons/poco/task/TaskDescriptionTests.java

```diff
@@ -375,14 +375,14 @@ void shouldNotBeEligibleToContributeAndFinalizeSinceWrongTrust() {
     }
 
     @Test
-    void shouldNotBeEligibleToContributeAndFinalizeSinceCallback() {
+    void shouldBeEligibleToContributeAndFinalizeEvenWithCallback() {
```

Comment:
What about moving this method below `shouldBeEligibleToContributeAndFinalize` ?
What do you think of trying to merge both test methods by using `@ParameterizedTest` instead of `@Test` ?

Reply:
Yes, I think it's the best idea because it will help eliminate redundant tests.